src/Nancy.Testing/DocumentWrapper.cs

```diff
@@ -19,9 +19,9 @@ public class DocumentWrapper
         /// <param name="buffer">The document represented as a byte array.</param>
         public DocumentWrapper(IEnumerable<byte> buffer)
         {
-            var utf8String = Encoding.UTF8.GetString(buffer.ToArray());
-
-            this.document = CQ.Create(utf8String);
+            var parser = new HtmlParser();
+            var stream = new MemoryStream(buffer.ToArray());

```

Comment:
I would place a using over these lines. Shouldn't matter much though as the stream will not keep an external handle alive.